v4-client-py-v2/dydx_v4_client/indexer/rest/modules/vaults.py

```diff
@@ -39,3 +39,15 @@ async def get_megavault_positions(self):
         """
         uri = f"/v4/vault/v1/megavault/positions"
         return await self.get(uri)
+
+    async def get_megavault_positions(self):
```

Comment:
I think we have this function implemented already (see the previous function), so feel free to remove this new implementation